test/unit/filters/RouteFilterSpec.scala

```diff
@@ -14,25 +14,48 @@
  * limitations under the License.
  */
 
+package unit.filters
+
 import akka.actor.ActorSystem
 import akka.stream.{ActorMaterializer, Materializer}
 import base.UnitSpec
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.{reset, when}
+import org.scalatest.BeforeAndAfterEach
 import org.scalatest.mockito.MockitoSugar
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
 import play.api.mvc.{RequestHeader, Result, Results}
 import play.api.test.FakeRequest
+import uk.gov.hmrc.customs.rosmfrontend.CdsErrorHandler
+import uk.gov.hmrc.customs.rosmfrontend.config.AppConfig
 import uk.gov.hmrc.customs.rosmfrontend.filters.RouteFilter
 
 import scala.concurrent.Future
 
-class RouteFilterSpec extends UnitSpec with MockitoSugar {
+class RouteFilterSpec extends UnitSpec with MockitoSugar with BeforeAndAfterEach with GuiceOneAppPerSuite {
```

Comment:
Why you added `GuiceOneAppPerSuite`? I can't see where you're using it

Reply:
You are right.  I copied the class definition from another "filters" test - forgot to remove that trait.